### ISSUE TITLE
kws_streaming demo training code seems not working well @rybakov

### ISSUE THREAD

**Issue by braveheartwithlove** [google-research/google-research#966]
@rybakov
I need some help here while running the 01_train.ipynb code, the dataset I used is from Speech Commands V2. I did not change any of your code.  While running this training part using ds_tc_resnet, the training log looks like this. Could you please help me check what is wrong on this? Thanks

==================================================================================================
Total params: 71,300
Trainable params: 69,764
Non-trainable params: 1,536
__________________________________________________________________________________________________
INFO:absl:Training from step: 1 
INFO:absl:Step #1: rate 0.001000, accuracy 0.00%, cross entropy 2.493576
INFO:absl:Step #2: rate 0.001000, accuracy 0.00%, cross entropy 2.628293
INFO:absl:Step #3: rate 0.001000, accuracy 0.00%, cross entropy 2.373230
INFO:absl:Step #4: rate 0.001000, accuracy 0.00%, cross entropy 2.624747
INFO:absl:Step #5: rate 0.001000, accuracy 0.00%, cross entropy 2.825552
INFO:absl:Step #6: rate 0.001000, accuracy 0.00%, cross entropy 2.542758
INFO:absl:Step #7: rate 0.001000, accuracy 0.00%, cross entropy 2.756225
INFO:absl:Step #8: rate 0.001000, accuracy 0.00%, cross entropy 2.711938
INFO:absl:Step #9: rate 0.001000, accuracy 0.00%, cross entropy 2.505342
INFO:absl:Step #10: rate 0.001000, accuracy 100.00%, cross entropy 2.063214
INFO:absl:Step #11: rate 0.001000, accuracy 0.00%, cross entropy 2.727900
INFO:absl:Step #12: rate 0.001000, accuracy 0.00%, cross entropy 2.802885
INFO:absl:Step #13: rate 0.001000, accuracy 0.00%, cross entropy 2.997224
INFO:absl:Step #14: rate 0.001000, accuracy 0.00%, cross entropy 2.247151
INFO:absl:Step #15: rate 0.001000, accuracy 0.00%, cross entropy 2.342353
INFO:absl:Step #16: rate 0.001000, accuracy 0.00%, cross entropy 2.332682

**Comment by braveheartwithlove** [comment]
Why the accuracy is 0.00%? I think it should keep increasing close to >95% as shown in your paper. What parameters should I change to reach a performance similar to your paper? Thanks!

**Comment by rybakov** [comment]
These colabs are used as fast playground, so most parameters are for toy example: e.g. FLAGS.batch_size=1. Please feel free to set it FLAGS.batch_size=100. I would suggest to use command line from [line](https://github.com/google-research/google-research/blob/master/kws_streaming/experiments/kws_experiments_12_labels.md#ds_tc_resnet---based-on-matchboxnet)

**Comment by rybakov** [comment]
With batch_size=100 training will report reasonable accuracy in the log you shared

**Comment by rybakov** [comment]
All experiments reported in the paper are described in [kws_experiments_paper_12_labels](https://github.com/google-research/google-research/blob/master/kws_streaming/experiments/kws_experiments_paper_12_labels.md). All colabs in this repo are toy examples.

**Comment by braveheartwithlove** [comment]
Thanks @rybakov!
